### ISSUE TITLE
Found a widget whose icon/text colors aren't changed? Report it here

### ISSUE THREAD

**Issue by luisbocanegra** [luisbocanegra/plasma-panel-colorizer#12]
Report those widgets here and I will try to add support to them **if possible**

**Requirements:**
- Icons must be symbolic (have no colors), non-symbolic icons can already be forced to have colors with "Force icon color" option

**Known:**
- [Fokus](https://store.kde.org/p/2117117)

**Comment by luisbocanegra** [comment]
Found some more OBS, screen casting and using microphone icons
![Screenshot_20240310_005323](https://github.com/luisbocanegra/plasma-panel-colorizer/assets/15076387/173a449d-1bcb-4548-a445-524295e1d6b8)

**Comment by baduhai** [comment]
Icons-only Task Manager and both the Pager and Activity Pager backgrounds aren't changed, is this on purpose?

**Comment by luisbocanegra** [comment]
Yes, they are blacklisted in the default configuration the reason is that those (and other widgets) create their own backgrounds and depending on the combination of both the color applied by this widget and the current color of the elements drawn by the widget they may become hard to see with some combinations.

A solution to avoid doing this could be somehow replacing those widget colors so they contrast with the current applied background but haven't looked into that yet.

**Comment by canlin-github** [comment]
After upgrade to Plasma 6.2, Steam tray icon stopped to colorized. In Plasma 6.1, I can force Steam icon color but now it's not in the widget list.
![Screenshot_20241011_203911](https://github.com/user-attachments/assets/407bd2f1-4aee-42c4-8e28-17a1df66b216)

**Comment by luisbocanegra** [comment]
Can reproduce after installing [v0.5.2](https://github.com/luisbocanegra/plasma-panel-colorizer/releases/tag/v0.5.2) but doesn't seem to be a problem on main branch, probably fixed as a side effect of the rewrite.

![Screenshot_20241011_084956](https://github.com/user-attachments/assets/061dc8eb-6d93-42ca-8629-70d3a62e9d33)

**Comment by luisbocanegra** [comment]
@canlin-github a new version [v1.0.0](https://github.com/luisbocanegra/plasma-panel-colorizer/releases/tag/v1.0.0) has been released, System Tray colorization should work fine with that version.

**Comment by canlin-github** [comment]
> @canlin-github a new version [v1.0.0](https://github.com/luisbocanegra/plasma-panel-colorizer/releases/tag/v1.0.0) has been released, System Tray colorization should work fine with that version.

Yes! It works great!
